### ISSUE TITLE
Configure pulseaudio application icon name

### ISSUE THREAD

**Issue by bbx0** [kpcyrd/spotify-launcher#12]
Spotify registers as pulseaudio stream with application name `spotify` but no application icon is shown within volume controls (e.g. pavucontrol).

An easy way to adjust is to provide the icon name directly via an [environment variable][1].
```sh
# Configure icon name via env
$ env PULSE_PROP_application.icon_name=spotify-launcher spotify-launcher
# Check the setting
$ pw-dump spotify |grep -F -e application.name -e application.icon-name
        "application.name": "spotify",
        "application.icon-name": "spotify-launcher",
```

This works within the DesktopEntry as well
```
[Desktop Entry]
Type=Application
Name=Spotify (Launcher)
GenericName=Music Player
Icon=spotify-launcher
TryExec=spotify-launcher
Exec=env PULSE_PROP_application.icon_name=spotify-launcher spotify-launcher %U
Terminal=false
MimeType=x-scheme-handler/spotify;
Categories=Audio;Music;Player;AudioVideo;
StartupWMClass=spotify
```

Would be nice to include the environment variable in the DesktopEntry to show the Spotify icon.

(This is different from #11, where the DesktopEntry is actually parsed for the icon name.)

[1]: https://www.freedesktop.org/wiki/Software/PulseAudio/Documentation/Developer/Clients/ApplicationProperties/#pa_prop_application_icon_name

**Comment by AidanGG** [comment]
Does my PR https://github.com/kpcyrd/spotify-launcher/pull/15 help here?

**Comment by bbx0** [comment]
Yes, adding `extra_env_vars = ["PULSE_PROP_application.icon_name=spotify-launcher"]` does resolve the issue. Just tested with a build based on your PR. :+1:
